src/ingestion-server/server/private/sg.ts

```diff
@@ -69,24 +71,8 @@ export function createALBSecurityGroup(
 export function createECSSecurityGroup(
   scope: Construct,
   vpc: IVpc,
-): SecurityGroup {
-  const ec2Sg = new SecurityGroup(scope, `${RESOURCE_ID_PREFIX}ecs-sg`, {
-    description: 'ECS security group',
-    vpc,
-    allowAllOutbound: true,
-  });
-
-  addCfnNagSuppressRules(ec2Sg.node.defaultChild as CfnResource, [
-    {
-      id: 'W40',
-      reason: 'Design intent: Security Groups egress with an IpProtocol of -1',
-    },
-    {
-      id: 'W5',
-      reason: 'Design intent: Security Groups found with cidr open to world on egress',
-    },
-  ]);
+): ISecurityGroup {
 
-  return ec2Sg;
+  return createSGForEgressToAwsService(scope, `${RESOURCE_ID_PREFIX}ecs-sg`, vpc);
```

Comment:
rename `ecs-sg` to `ECSEgressToAWSServiceSG`

Reply:
changed